protostar/protostar_toml/_conftest.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any
```

Comment:
Test discovery didn't work in VS Code. `conftest.py` caused weird import issues. I needed a file to store a test utils rather than fixtures, so I renamed this file.

Reply:
`conftest.py` caused weird import issues wdym by that?

Reply:
In the VS Code output panel other there were "coulnd't import ..." in other `conftest.py` files.

Reply:
```
__________________________________________________________________ ERROR collecting protostar/cli/argument_parser_facade_test.py __________________________________________________________________
ImportError while importing test module '/protostar/protostar/cli/argument_parser_facade_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../.pyenv/versions/3.7.12/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
protostar/cli/argument_parser_facade_test.py:7: in <module>
    from conftest import BaseTestCommand, FooCommand
E   ImportError: cannot import name 'BaseTestCommand' from 'conftest' (/protostar/protostar/protostar_toml/conftest.py)
_________________________________________________________________________ ERROR collecting protostar/cli/cli_app_test.py __________________________________________________________________________
ImportError while importing test module '/protostar/protostar/cli/cli_app_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../.pyenv/versions/3.7.12/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
protostar/cli/cli_app_test.py:6: in <module>
    from conftest import FooCommand
E   ImportError: cannot import name 'FooCommand' from 'conftest' (/protostar/protostar/protostar_toml/conftest.py)
```

Reply:
I had to create `__init__.py`. Imports in Python are so unintuitive.

![Screen Shot 2022-06-15 at 16 52 57](https://user-images.githubusercontent.com/9629577/173858936-09296b7e-53a8-491a-8320-ad91c5507126.png)